polkadot/runtime/common/src/slots/mod.rs

```diff
@@ -118,8 +144,22 @@ pub mod pallet {
 	pub type Leases<T: Config> =
 		StorageMap<_, Twox64Concat, ParaId, Vec<Option<(T::AccountId, BalanceOf<T>)>>, ValueQuery>;
 
+	/// Information about the currently leased slot of the leaser.
+	///
+	/// Keyed by para_id and leaser, value is a tuple of current_reserved_amount and
+	/// lease_period_count for the latest lease.
```

Comment:
```suggestion
	/// Keyed by `ParaId` and leaser, value is a tuple of `current_reserved_amount` and
	/// `lease_period_count` for the latest lease.
```

or maybe just have the whole text without vars.

Reply:
Yeah good point, made them as whole texts.